### PR TITLE
[Code Enchancement] Support for ovos config

### PR DESCRIPTION
**Pull request by NeonKirill**

### CODE REVIEW

**Comment by NeonKirill** [comment]
@NeonDaniel please review it, but overall I think I will have to split up pyklatchat modules, so it will be easier to manage

File: chat_server/server_utils/auth.py [hunk 1]
```diff
@@ -32,8 +32,8 @@
 import jwt
 
 from time import time
-from fastapi import Response, Request
-from neon_utils import LOG
+from fastapi import Request
+from ovos_utils import LOG
```
Comment (NeonDaniel) [inline chat_server/server_utils/auth.py:36]: Should this be `utils.logging_utils` like other imports?

File: chat_server/sio.py [hunk 2]
```diff
@@ -34,8 +34,8 @@
 from time import time
 from typing import List, Optional
 
-from neon_utils import LOG
-from neon_utils.cache_utils import LRUCache
+from cachetools import LRUCache
+from ovos_utils import LOG
```
Comment (NeonDaniel) [inline chat_server/sio.py:38]: Should this be utils.logging_utils like other imports?

File: docker_overlay/etc/neon/chat_server.yml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+      SECRET: "123"
+      JWT_ALGO: "HS256"
+    FILE_STORING_TYPE: "sftp"
+    FILE_STORING_LOCATION: "/app/static"
+    LIBRE_TRANSLATE_URL: "https://libretranslate.2022.us"
+    "SFTP":
+      HOST": "127.0.0.1",
+      USERNAME": "klat_server",
+      PASSWORD": "123",
+      PORT": 22,
```
Comment (NeonDaniel) [inline docker_overlay/etc/neon/chat_server.yml:37]: Looks like there are some leftover `"` characters in config keys here

File: docker_overlay/etc/neon/chat_server.yml [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
```
Comment (NeonDaniel) [inline docker_overlay/etc/neon/chat_server.yml:55]: Looks like there are some leftover `"` characters in config keys here

**Comment by NeonDaniel** [comment]
I see the automation is failing

```
An image does not exist locally with the tag: ghcr.io/neongeckocom/pyklatchat_base
```

File: utils/logging_utils.py [hunk 5]
```diff
@@ -0,0 +1,7 @@
+import importlib
+import logging
+
```
[the remark this answers is not in the document]
Comment (NeonKirill) [reply utils/logging_utils.py:31]: > Any reason this isn't just `from ovos_utils.log import LOG`? That also handles the log overrides from config so the `combo_lock` patching shouldn't be necessary (just addded to default config)

I see that in config we only set certain level of those, while I want to disable it completely...

[the remark this answers is not in the document]
Comment (NeonDaniel) [reply utils/logging_utils.py:31]: I think errors or at least exceptions should be shown to debug failures

**Review by NeonDaniel — APPROVED** [review]

File: utils/logging_utils.py (hunk 5 above)
[the remark this answers is not in the document]
Comment (NeonKirill) [reply utils/logging_utils.py:31]: > I think errors or at least exceptions should be shown to debug failures

Okay, I think I will leave it as is for now, and focus on implementing the features Elon and Richard are asking me for, and will get back to that later

**Pull request merged**